packages/shared/src/components/squads/settings/PublicStatusPanel.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from 'react';
+import { Button, ButtonVariant } from '../../buttons/Button';
+import { StatusDescription } from './common';
+import { ProgressBar } from '../../fields/ProgressBar';
+
+interface PublicStatusPanelProps {
+  count: number;
+  required: number;
+}
+
+export function PublicStatusPanel({
+  count,
+  required,
+}: PublicStatusPanelProps): ReactElement {
+  return (
+    <div className="flex flex-col items-end gap-2">
+      <StatusDescription className="flex w-full">
+        <span className="mr-auto">Posts</span>
+        <strong>{count}/3</strong>
+      </StatusDescription>
+      <span className="flex w-full overflow-hidden rounded-16 bg-accent-pepper-subtler">
```

Comment:
Yeah, was thinking of that too, to make it optional. Maybe we can add it as a prop to include the bg since from the previous implementations, it wasn't needed.

Reply:
Fixed it here since I already touched the component itself 🚀